src/sdk/SpeechTranslationConfig.ts

```diff
@@ -43,6 +43,9 @@ export abstract class SpeechTranslationConfig extends SpeechConfig {
      *       expires, the caller needs to refresh it by setting the property authorizationToken with a new
      *       valid token. Otherwise, all the recognizers created by this SpeechTranslationConfig instance
      *      will encounter errors during recognition.
+     * As configuration values are copied when creating a new recognizer, the new token value will not apply to recognizers that have already been created.
```

Comment:
Indentation?

Reply:
this was on purpose since this is a separate topic

Reply:
fixed